linter_exclusions.yml

```diff
@@ -1752,16 +1770,25 @@ eventgrid system-topic event-subscription create:
     preferred_batch_size_in_kilobytes:
       rule_exclusions:
       - option_length_too_long
+    delivery_attribute_mapping:
+      rule_exclusions:
+      - option_length_too_long
 eventgrid system-topic event-subscription show:
   parameters:
     include_full_endpoint_url:
       rule_exclusions:
       - option_length_too_long
+    include_static_delivery_attribute_secret:
+      rule_exclusions:
+      - option_length_too_long
```

Comment:
I think it's too long. Why not add a short option for it?

Reply:
It matches the rest api property and a pattern we already have in the CLI.  Could not find anything useful as a shortcut.

Reply:
How about `attrib_secret` as a shortcut?

Reply:
have another option for this called --include-attrib-secret